tenseal/cpp/context/tensealcontext.h

```diff
@@ -27,8 +33,6 @@ class TenSEALContext {
      * time for doing operations on encrypted tensors since we only need to
      * instantiate them once.
      **/
-    shared_ptr<Encryptor> encryptor;
-    shared_ptr<Decryptor> decryptor;
```

Comment:
The reason that we kept these objects in the beginning was because we didn't want to recreate them everytime we needed them. I see that you dropped the decryptor, is there any reason for that?

Reply:
The decryptor wasn't used anywhere. We already recreate it from a secret key, local or provided.